### DISCUSSION TITLE
Comparison to other open source VRP projects

### DISCUSSION THREAD

**Discussion by do-me** [PyVRP/PyVRP#578] [General]
Hi folks, 
first of all: cool to see more people working in the VRP-space! :)

I just went through the issues and discussions and was surprised that noone else was asking about comparisons to existing solutions, particularly: 
- [VROOM](https://github.com/VROOM-Project/vroom) and 
- [VRP](https://github.com/reinterpretcat/vrp)

I'm a big fan of both and would like to know where this project sees it's strengths in comparison. Did you invent new heuristics or similar? It would be great if you could add something about your motivation/goals in the readme.

As you're also developing in C++ it naturally also begs the question why not to join forces with or build partially on VROOM?

**Comment by N-Wouda** [comment]
Hi @do-me,

Our main goal is to make the performance of existing (academic) algorithmic ideas available to the wider community through an easy-to-use Python package. To achieve this, we are primarily focused on performance over somewhat long runs; e.g., after a few minutes to hours. Extensibility is in scope, and we do work towards supporting additional VRP variants, but also take care not to have such extensions hurt performance too much.

We have a benchmark comparison with OR-Tools and other (academic) solvers in our documentation, [here](https://pyvrp.org/setup/benchmarks.html). We may at some point also add a comparison with VROOM, but haven't gotten around to that yet.

>As you're also developing in C++ it naturally also begs the question why not to join forces with or build partially on VROOM?

Largely because this is a Python package with a large C++ component, not a fully C++ codebase. Also because VROOM's really focused on getting solutions really quickly, whereas PyVRP aims to get near-optimal solutions over longer runtimes (at least with its default parameter configuration).

**Reply by do-me** [reply]
Thanks for your explanations. Really curious to give it a try soon!
